tests/compile-fail/concurrency/dangling_tls_lib.rs

```diff
@@ -0,0 +1,46 @@
+// ignore-windows
```

Comment:
Why is this ignored on Windows?

Reply:
Oh because we can't spawn threads there. Please explain that in a comment like we do elsewhere for `ignore`s.